cpanspec

```diff
@@ -1576,6 +1551,46 @@ END
         close($tfh);
     }
 }
+}
+
+sub generate_autosetup {
+    my ($config) = @_;
+    my @patches;
+    my @patch_sources;
+    my $all_patch_args_same = 1;
+    my $autosetup_args = '-p1';
+    my %spec = (autosetup_args => '-p1', patches => \@patches, sources => \@patch_sources);
+    return \%spec unless $config->{patches};
+
+    for my $patch (sort keys %{ $config->{patches} }) {
+        my $args = $config->{patches}->{ $patch } // '';
```

Comment:
why defaults to empty string when it was checked previously for existence(L1563)? if there a chance `$config->{patches}` is defined but `$config->{patches}->{ $patch }` is not?